### PR TITLE
New UI: dashboard header no longer uses cookie, fix tips

### PR DESCRIPTION
**Pull request by breville**
The dashboard header now uses current_user instead of the user_type cookie, which improves things so that the first time an already signed-in user sees the new UI they see the correct header/hamburger options.  Prior to this change, the user would see the options for a signed-out user until their next request, which would have the new user_type cookie for the first time.

For dashboard pages where current_user is `nil`, we still fall back to the cookie.

Also fixes the /home tip for the hamburger to point directly at it.

Here's what an already signed-in user should now see the first time they see the new UI:

screenshot 2017-05-31 13 02 35

### CODE REVIEW

**Comment by tanyaparker** [comment]
How come there's no "Hi Brendan" button in your screenshot?

**Comment by tanyaparker** [comment]
And the X to close the tooltips are missing from your screenshot?

**Comment by breville** [comment]
Those were the results of a very slow local server which hadn't yet finished rendering the page.  Screenshot updated.

File: dashboard/app/views/layouts/_header.html.haml [hunk 1]
```diff
@@ -16,8 +16,15 @@
 
 - if show_header2
   %link{:href=>'/shared/css/hamburger.css', :rel=>'stylesheet'}
-  - cookie_key = '_user_type' + (Rails.env.production? ? '' : "_#{Rails.env}")
-  - user_type = request.cookies[cookie_key]
+  - if current_user
+    - if current_user.teacher?
+      - user_type = "teacher"
+    - elsif current_user.student?
+      - user_type = "student"
```
Comment (wjordan) [inline dashboard/app/views/layouts/_header.html.haml:23]: this could be simplified to `user_type = current_user.user_type`

**Comment by tanyaparker** [comment]
👍

**Pull request merged**